beneficiaries.yaml

```diff
@@ -26,7 +26,7 @@ components:
       description: A beneficiary entity type.
       enum:
         - individual
-        - business
+        - corporate
       x-enum-varnames:
         - individual
         - business
```

Comment:
@pateluday24 what about this field?
does this also need to change to `corporate`?

Reply:
enum  and x-enum-varnames are 2 different things. The varnames represent the varible name that stores the enum value. So here the value "corporate" will be represented by "business" in the code and for externally it will be shown as "corporate"

Reply:
Why do we want the enum to be represented as "business" in the code if it's externally represented as "corporate"?

Reply:
To have less BE code changes but either ways I am ok to drop it.

Reply:
I think it would be better to drop business since it's a long term fix. It would also remove further need to convert between business and corporate